app/models/claim/search.rb

```diff
@@ -0,0 +1,31 @@
+class Claim
+  class Search
+    attr_accessor :query
+
+    SEARCHABLE_ATTRIBUTES = %w[
+      email_address
+      surname
+      teacher_reference_number
+    ]
+
+    def initialize(query)
+      @query = query
+    end
+
+    def claims
+      claims = search_by(:reference)
+
+      SEARCHABLE_ATTRIBUTES.each do |attribute|
+        claims = claims.or(search_by(attribute))
+      end
+
+      claims
```

Comment:
It made me uncomfortable that `reference` wasn't in `SEARCHABLE_ATTRIBUTES`. I think we can rewrite this method such that we can put it there:

```suggestion
      SEARCHABLE_ATTRIBUTES.inject(Claim.none) do |relation, attribute|
        relation.or(search_by(attribute))
      end
```

Reply:
Yes! I forgot about `.none`. Good shout